lib/ClimaCoreVTK/test/runtests.jl

```diff
@@ -261,4 +281,44 @@ end
         basis = :point,
     )
     @test isfile(joinpath(OUTPUT_DIR, "hybrid_sphere_point_latlong.vtu"))
+
+    writepvd(
+        joinpath(OUTPUT_DIR, "series", "hybrid_sphere_scalar_cell"),
+        times,
+        Fields.coordinate_field(cspace);
+        basis = :cell,
+    )
```

Comment:
@simonbyrne I know this is not urgent, but just to follow-up on our recent conversation, the 3d-sphere time-series vtk outputs don't seem to work with either `basis = :cell` or `basis = :point`. Is it supposed to work with all except for `:lagrange`? 

The error is a MethodError on `getindex`
```
MethodError: no method matching getindex(::ClimaCore.Fields.Field{ClimaCore.DataLayouts.VIJFH{ClimaCore.Geometry.LatLongZPoint{Float64}, 5, SubArray{Float64, 5, Array{Float64, 5}, Tuple{Base.Slice{Base.OneTo{Int64}}, Base.Slice{Base.OneTo{Int64}}, Base.Slice{Base.OneTo{Int64}}, UnitRange{Int64}, Base.Slice{Base.OneTo{Int64}}}, false}}, ClimaCore.Spaces.ExtrudedFiniteDifferenceSpace{ClimaCore.Spaces.CellCenter, ClimaCore.Spaces.SpectralElementSpace2D{ClimaCore.Topologies.Topology2D{ClimaCore.Meshes.EquiangularCubedSphere{ClimaCore.Domains.SphereDomain{Float64}, ClimaCore.Meshes.NormalizedBilinearMap}, CartesianIndices{3, Tuple{Base.OneTo{Int64}, Base.OneTo{Int64}, Base.OneTo{Int64}}}, LinearIndices{3, Tuple{Base.OneTo{Int64}, Base.OneTo{Int64}, Base.OneTo{Int64}}}, NamedTuple{(), Tuple{}}}, ClimaCore.Spaces.Quadratures.GLL{5}, ClimaCore.Geometry.SphericalGlobalGeometry{Float64}, ClimaCore.DataLayouts.IJFH{ClimaCore.Geometry.LocalGeometry{(1, 2), ClimaCore.Geometry.LatLongPoint{Float64}, Float64, StaticArraysCore.SMatrix{2, 2, Float64, 4}}, 5, Array{Float64, 4}}, ClimaCore.DataLayouts.IJFH{Float64, 5, Array{Float64, 4}}, ClimaCore.DataLayouts.IFH{ClimaCore.Geometry.SurfaceGeometry{Float64, ClimaCore.Geometry.UVVector{Float64}}, 5, Array{Float64, 3}}, NamedTuple{(), Tuple{}}}, ClimaCore.Topologies.IntervalTopology{ClimaCore.Meshes.IntervalMesh{ClimaCore.Domains.IntervalDomain{ClimaCore.Geometry.ZPoint{Float64}, Tuple{Symbol, Symbol}}, LinRange{ClimaCore.Geometry.ZPoint{Float64}, Int64}}, NamedTuple{(:bottom, :top), Tuple{Int64, Int64}}}, ClimaCore.Spaces.Flat, ClimaCore.Geometry.SphericalGlobalGeometry{Float64}, ClimaCore.DataLayouts.VIJFH{ClimaCore.Geometry.LocalGeometry{(1, 2, 3), ClimaCore.Geometry.LatLongZPoint{Float64}, Float64, StaticArraysCore.SMatrix{3, 3, Float64, 9}}, 5, Array{Float64, 5}}, ClimaCore.DataLayouts.VIJFH{ClimaCore.Geometry.LocalGeometry{(1, 2, 3), ClimaCore.Geometry.LatLongZPoint{Float64}, Float64, StaticArraysCore.SMatrix{3, 3, Float64, 9}}, 5, Array{Float64, 5}}}}, ::Int64)
```
happening here on line 285. 

Did I do something wrong that I am missing in the test? 
To reproduce simply run `include("lib/ClimaCoreVTK/test/runtests.jl")`